scripts/PublicApi/mark-shipped.ps1

```diff
@@ -7,6 +7,10 @@ $ErrorActionPreference = "Stop"
 function MarkShipped([string]$dir) {
     $shippedFilePath = Join-Path $dir "PublicAPI.Shipped.txt"
     $shipped = Get-Content $shippedFilePath
+    if ($null -eq $shipped) {
+        $shipped = @()
+    }
```

Comment:
... how do you get into this case though. The `Get-Content` function should return `$null` only in error cases and that should throw.

Reply:
For new projects, such as CodeStyle. The PublicAPI.Shipped.txt files are empty, and `Get-Content` returns null.

Reply:
That's terrifying ...